src/runtime/runtime.ts

```diff
@@ -144,6 +178,12 @@ class Duomo implements Runtime {
 		return this.#darkMode
 	}
 	setDarkMode(mode: boolean) {
+		const durValue = getComputedStyle(document.documentElement).getPropertyValue('--default-theme-transition-duration');
+		
+		if(this.#shouldNoOpDarkMode){ 
```

Comment:
Make sure you’re using Prettier if you’re not already. I would expect this to be `if (this.#shouldNoOpDarkMode) {` (you have no spaces right now).

Reply:
I installed the Prettier, but It's still same. Probably prettier setting problem on my vscode..